types/time.go

```diff
@@ -886,6 +891,7 @@ func GetTimezone(lit string) (idx int, tzSign, tzHour, tzSep, tzMinute string) {
 	return -1, "", "", "", ""
 }
 
+// splitDateTime
```

Comment:
The public function has to add this comment. you can remove it.

ditto other.

Reply:
If don't add these kind of comment, goword check will err, indicating spelling errors.
![image](https://user-images.githubusercontent.com/44599853/150759907-36c6ff84-a1b5-4e26-a0a8-9c6cf76fc0bf.png)